WishList/WishList/Friends/SearchFriendsScreen.swift

```diff
@@ -38,20 +38,33 @@ struct SearchFriendsScreen: View {
         }.searchable(text: $searchText, placement: .navigationBarDrawer(displayMode: .always), prompt: "Search friends")
             .overlay{
                 Button(action: {
-                    performSearch("")
+                    searchFriend()
```

Comment:
This is ok for now, but let's keep in mind this won't be the final behaviour. This method should be called when the user press the enter key on the keyboard.